README.md

```diff
@@ -1,46 +1,76 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Ez Car
+# Ez Car Backend
 
-> This project contains an API that is consumed by our React frontend. It was built using Rails as an API.
+> This is an API allows users to book a car present in our database. The user can set the reservation date and final date for a selected car. If the user is an administrator they can upload a car and delete it from our database.
 
 ## ERD diagram
 
 ![ERD](https://user-images.githubusercontent.com/97900045/196997928-2c857de9-a9fc-4216-8480-9580cc9bc6c0.png)
 
 ## Built With
 
+### Technologies 
+
 - Ruby on Rails
 - PostgreSQL
 - RSpec
 - Rswag
+- Devise 
+- Devise-JWT
+- CanCanCan
+- Capybara
 
-## Live Demo (if available)
+### Tools
 
-[Live Demo Link](https://livedemo.com)
+- Visual Studio Code
+- GIT
+- Rubocop
+- Heroku 
 
+## Live Demo (if available)
 
-## Getting Started
+- [eZ_car](https://ezcar-backend.herokuapp.com/)
+- [Video for the API documentation](https://www.loom.com/share/c4df2507582d4538ae3755f26e359c7f)
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
+## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+- ruby 
+- rails 
+- git 
+- bundle
 
 ### Setup
 
+#### Open Terminal 
+
+    git clone git@github.com:indigodavid/ezCar_backend.git
+    cd ezCar_backend
+
 ### Install
 
+    bundle install
+    rails db:create
+    rails db:migrate
+    rails db:seed
+
 ### Usage
 
+    rails server
+    
 ### Run tests
 
+    rspec spec
```

Comment:
- [ ] Upon calling `rspec spec` according to your instructions, some of the tests are presented as failures, because it calls the whole test suite, including the rswag ones:

![image](https://user-images.githubusercontent.com/57421823/199336705-91524980-39b2-4181-a00c-32b17fe9e194.png)

## Tips and extra help

Therefore, instead of just `rspec spec` please, consider using these 2 commands in your readme instructions instead:

`rspec spec/models`

and

`rake rswag`

Result:

![image](https://user-images.githubusercontent.com/57421823/199336308-3628b27f-6909-4bbb-8465-d79419145f93.png)

And there you go 19 examples and 0 failures!

That way, your users will never get a failed example 💪